.github/workflows/tests.yml

```diff
@@ -48,9 +48,14 @@ jobs:
       - name: Run Homebrew/homebrew-test-bot RSpec tests
         run: bundle exec rspec
 
-      - name: Build Docker image
+      - name: Build Docker image and start container
```

Comment:
Can you provide more context for all the Docker changes? They seem unrelated but I presume they are not.

Reply:
The `--only-formulae-dependents` test did nothing because we weren't passing `--testing-formulae`.

When I added it, it revealed that because we're creating a new container for each test then it errored. `--only-formulae-dependents` assumes that the formula is already installed (which can be changed but I think makes sense).

Reply:
Gotcha, thanks ❤️